README.md

```diff
@@ -0,0 +1,50 @@
+[![Build Status][ci-img]][ci] [![Released Version][maven-img]][maven]
+
+# OpenTracing Apache HttpClient 
+
+Apache HttpClient instrumentation.
+
+## Configuration
+
+```java
+GlogalTracer.register(tracer);
+HttpClient httpClient = new TracingHttpClientBuilder()
+    .build();
+
+//custom configuration
+HttpClient httpClient = new TracingHttpClientBuilder(
+        redirectStrategy, 
+        redirectHandlingDisabled,
+        GlobalTracer.get(),
+        spanDecorators)
+    .build();
+```
+
+## Linking to parent span
+If parent span context is not passed created client spans will be in a new trace.
+
+### SpanManager
+```java
+spanManager.activate(parentSpan);
```

Comment:
Add a comment with the limitations. For instance, I believe this will not work properly if people are making async calls, as it's then executed in separate threads/span contexts.